newsfragments/5134.fixed.md

```diff
@@ -0,0 +1 @@
+Fix clippy errors
```

Comment:
You don't need a news fragment for simple refactors and fixes like this. I'll add the label so you can delete this file.